angular.json

```diff
@@ -86,8 +88,15 @@
             "polyfills": "src/polyfills.ts",
             "tsConfig": "tsconfig.spec.json",
             "karmaConfig": "karma.conf.js",
-            "assets": ["src/favicon.ico", "src/assets"],
-            "styles": ["src/styles.scss", "src/assets/scss/main.scss"],
+            "assets": [
+							"src/favicon.ico", 
+							"src/assets"
+						],
+            "styles": [
+							"src/styles.scss", 
```

Comment:
We actually don't need styling while testing. This allows for faster testing.

Reply:
Fixed, @bl4ck4ndbr0wn